### ISSUE TITLE
[reproducibility] When evaluating, the test miou stays at around 0.1 while the training miou improves

### ISSUE THREAD

**Issue by AlbertiPot** [facebookresearch/unnas#8]
Hi, thanks for your bravo works.

However, when I evaluate with the search_based/eval_phase/seg/cityscapes_seg.yaml, the batch size and the learning rate are all the same with the config specified in yaml. However, the training miou improves to 0.83 at epoch 2700, but the test miou stays at around 0.1 through out the training. We use 8 A100 GPUs and the pytorch is 1.8.1 with python3.8
